src/jsMain/kotlin/io/beatmaps/shared/sharedComponents.kt

```diff
@@ -216,6 +215,31 @@ val playlistOwner = fc<PlaylistOwnerProps> { props ->
     }
 }
 
+external interface ReviewerProps : Props {
```

Comment:
Am I missing the reason this wasn't just a new property on the existing component?

Reply:
I figured splitting them would be cleaner. You'd never use that property on playlists, or anywhere other than reviews. I can merge them again if you want though, just say the word.

Reply:
It's not a big deal, just results in duplicating the rest of the code. Probably a small enough component to not worry about too much.